rpc/types/src/misc/distribution.rs

```diff
@@ -109,7 +118,16 @@ epee_object! {
 pub struct DistributionCompressedBinary {
     pub start_height: u64,
     pub base: u64,
-    pub compressed_data: String,
+    #[cfg_attr(
+        feature = "serde",
+        serde(serialize_with = "serialize_distribution_as_compressed_data")
+    )]
+    #[cfg_attr(
+        feature = "serde",
+        serde(deserialize_with = "deserialize_compressed_data_as_distribution")
+    )]
+    #[cfg_attr(feature = "serde", serde(rename = "compressed_data"))]
+    pub distribution: Vec<u64>,
```

Comment:
`distribution` is now always uncompressed from the constructor's perspective.

It gets (de)compressed during (de)serialization in `epee/serde`.